packaging/irods_prc_tests/test_rule_engine_plugin_metadata_guard_atomic.py

```diff
@@ -124,5 +124,22 @@ def test_guard_atomic_operations_editor_list__issue_38(self):
             obj.metadata.apply_atomic_operations( AVUOperation(operation='add', avu=iRODSMeta('irods::editorlist', 'editorlist')))
             self.assertEqual(obj.metadata['irods::editorlist'], iRODSMeta('irods::editorlist', 'editorlist'))
 
+        # remove test user from editor list
+        with iRODSSession(host='localhost', port=1247, user='rods', password='rods', zone='tempZone') as local_admin:
+            coll = local_admin.collections.get("/tempZone")
+            METADATA_GUARD_CONFIG = {
+                'admin_only': False,
+                'editors': [{'name': 'rods', 'type': 'user'}],
+                'prefixes': ['irods::']
+            }
+            coll.metadata['irods::metadata_guard'] = iRODSMeta('irods::metadata_guard', json.dumps(METADATA_GUARD_CONFIG))
+
+        with iRODSSession(host='localhost', port=1247, user='alice', password='test', zone='tempZone') as unprivileged_user:
+            obj = unprivileged_user.data_objects.get(self.TEST_DATA_OBJECT_PATH)
+
+            # operation should succeed, as alice is set as an editor
```

Comment:
Is this comment correct?

Reply:
agreed.   i'm reading the code to say that alice is NOT an editor, and we're asserting a failure in L141 just below.